docker/Dockerfile.db

```diff
@@ -0,0 +1,11 @@
+FROM postgis/postgis:10-2.5
```

Comment:
Why Postgres10? We are running postgres13 in production now, and latest stable version is 14.1

Reply:
Latest postgis is 3.2

Reply:
I got it from here:

```
ivan@epigraph:~$ dpkg -l| grep post
ii  postgis                            2.4.3+dfsg-4                             amd64        Geographic objects support for PostgreSQL
ii  postgresql                         10+190ubuntu0.1                          all          object-relational SQL database (supported version)
ii  postgresql-10                      10.19-0ubuntu0.18.04.1                   amd64        object-relational SQL database, version 10 server
ii  postgresql-10-plr                  1:8.3.0.17-3                             amd64        Procedural language interface between PostgreSQL and R
ii  postgresql-10-postgis-2.4          2.4.3+dfsg-4                             amd64        Geographic objects support for PostgreSQL 10
ii  postgresql-10-postgis-2.4-scripts  2.4.3+dfsg-4                             all          Geographic objects support for PostgreSQL 10 -- SQL scripts
ii  postgresql-10-postgis-scripts      2.4.3+dfsg-4                             all          transitional dummy package
ii  postgresql-client-10               10.19-0ubuntu0.18.04.1                   amd64        front-end programs for PostgreSQL 10
ii  postgresql-client-common           190ubuntu0.1                             all          manager for multiple PostgreSQL client versions
ii  postgresql-common                  190ubuntu0.1                             all          PostgreSQL database-cluster manager
ii  postgresql-contrib                 10+190ubuntu0.1                          all          additional facilities for PostgreSQL (supported version)
ii  postgresql-plpython-10             10.19-0ubuntu0.18.04.1                   amd64        PL/Python procedural language for PostgreSQL 10
ii  postgresql-plpython3-10            10.19-0ubuntu0.18.04.1                   amd64        PL/Python 3 procedural language for PostgreSQL 10
```

is there another place with the production postgres installation?

Reply:
Well, I think It might have been from the previous setup before I switch to Ansible. 
We can, nevertheless install the latest versions if we add the postrges apt repository to our `sources.list` As shown [here](https://citizix.com/how-to-install-and-configure-postgres-14-ubuntu-20-04/?utm_source=rss&utm_medium=rss&utm_campaign=how-to-install-and-configure-postgres-14-ubuntu-20-04)

Reply:
ok ... so I will pin it to postgres 14 and postgis  to 3.2
the latest versions available for docker

Reply:
> > and the forecast 4 (that populates `df_for_hosp_cantons`) never ended ... maybe it is because my machine is not good at all ...
> 
> This code can benefit from some optimization. I have asked @eduardocorrearaujo to look into that.

nice! I created an issue about that, I will assign it to him. thanks!